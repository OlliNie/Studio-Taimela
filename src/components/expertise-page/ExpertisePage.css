.expertisePage{
  display: flex;
  box-sizing: border-box;
  flex-direction: column;

  padding: 8rem 1rem 0 1rem;
  text-align: center;
  min-height: calc(100vh - 10rem);
  background-image: url('../../assets/images/mountain.png');
  background-repeat: no-repeat;
  background-size: contain;
  background-position: bottom;

  @media (min-width: 750px){
  padding: 10rem  5rem 0 5rem;
  }
  @media(min-width:750px) and (min-height:950px){
    background-size: 120% 800px;
  }
  @media(min-width:900px){
    background-size: 100% auto ;
  }
  @media(min-width:900px) and (min-height:1100px){
    background-size: 120% 1400px;
  }
  @media (min-width: 1100px){
    min-height: calc(100vh - 0rem);
    padding: 12rem  5rem 0 5rem; 
    background-position: left -4px bottom -50px;
  }
}




