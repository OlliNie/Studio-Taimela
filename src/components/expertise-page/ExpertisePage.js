import React from 'react';
import ExpertiseList from './ExpertiseList';
import styles from './ExpertisePage.css';
import Title from '../title/Title';
import ContentDevider from '../content-devider/ContentDevider';

export default function ExpertisePage() {
  return (
    <>
      <section
        id="Expertise"
        className={[styles.expertisePage, 'expertisePage'].join(' ')}
      >
        <Title title="Expertise" />
        <ExpertiseList />
      </section>
      <ContentDevider />
    </>
  );
}
