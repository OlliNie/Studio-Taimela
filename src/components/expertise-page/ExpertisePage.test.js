import React from 'react';
import { shallow } from 'enzyme';
import ExpertisePage from './ExpertisePage';

describe(`ExpertisePage component`, () => {
  it('renders Expertise', () => {
    const wrapper = shallow(<ExpertisePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
