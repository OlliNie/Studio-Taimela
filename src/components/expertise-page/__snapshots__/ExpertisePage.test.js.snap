// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpertisePage component renders Expertise 1`] = `
<Fragment>
  <section
    className="expertisePage expertisePage"
    id="Expertise"
  >
    <Title
      title="Expertise"
    />
    <ExpertiseList />
  </section>
  <SubSection />
</Fragment>
`;
